xbmc/addons/interfaces/gui/General.cpp

```diff
@@ -212,6 +215,66 @@ ADDON_HARDWARE_CONTEXT Interface_GUIGeneral::get_hw_context(KODI_HANDLE kodiBase
   return CServiceBroker::GetWinSystem()->GetHWContext();
 }
 
+void Interface_GUIGeneral::get_whitelist_resolutions(KODI_HANDLE kodiBase,
+                                                     int** widthList,
+                                                     int** heightList,
+                                                     float** refreshRateList,
+                                                     unsigned int* size)
+{
```

Comment:
done,
i found that `Debug assertion failed _crtisValidHeapPointer(block)` error on `free` call is raised because i build ISAdaptive as debug and Kodi as release
for some reason is influenced by this